pkg/partmgr/partmgr.go

```diff
@@ -97,7 +102,7 @@ func (p *PartitionManager) Init() error {
 }
 
 func (p *PartitionManager) TimeToPart(t int64) (*DBPartition, error) {
-	interval := IntervalInMilli(p.cfg.PartitionSchemaInfo.PartitionerInterval)
+	interval, _ := utils.Str2duration(p.cfg.PartitionSchemaInfo.PartitionerInterval)
```

Comment:
@dinal rather not do string parsing on fast path, need to be taken from the priv variables